src/JWT/IJwtDecoder.cs

```diff
@@ -65,6 +94,17 @@ public interface IJwtDecoder
         /// <exception cref="SignatureVerificationException">Thrown if the verify parameter was true and the signature was NOT valid or if the JWT was signed with an unsupported algorithm.</exception>
         /// <exception cref="TokenExpiredException">Thrown if the verify parameter was true and the token has an expired exp claim.</exception>
         IDictionary<string, object> DecodeToObject(string token, byte[] key, bool verify);
+        
+        /// <summary>
+        /// Given a JWT, decodes it and return the payload as an object.
+        /// </summary>
+        /// <param name="token">The JWT.</param>
+        /// <param name="keys">The key which one of them was used to sign the JWT.</param>
+        /// <param name="verify">Whether to verify the signature (default is true).</param>
+        /// <returns>An object representing the payload.</returns>
+        /// <exception cref="SignatureVerificationException">Thrown if the verify parameter was true and the signature was NOT valid or if the JWT was signed with an unsupported algorithm.</exception>
+        /// <exception cref="TokenExpiredException">Thrown if the verify parameter was true and the token has an expired exp claim.</exception>
+        IDictionary<string, object> DecodeToObject(string token, List<byte[]> keys, bool verify);

```

Comment:
`IReadOnlyCollection<byte[]>` here too please, and elsewhere too. Please search for all entries of `List<>`